action-api/src/main/java/com/powsybl/action/PhaseTapChangerTapPositionAction.java

```diff
@@ -30,4 +32,12 @@ public PhaseTapChangerTapPositionAction(String id, String transformerId, boolean
     public String getType() {
         return NAME;
     }
+
+    public NetworkModification toModification() {
+        // did not use PhaseShifterShiftTap in case of relative change because PhaseShifterShiftTap change also regulating and its mode
+        // so update PhaseTapPositionModification with is relative instead
```

Comment:
These 2 lines of comment can be deleted.

Reply:
I am not sure to fully understand it:
If you suggest to create 2 actions and 2 modifications, 1 of each for each type of transformer (2w and 3w). I don't think this is a good idea since there's no distinction on tap changers nowadays.
If it 's only to stress that the side should only be filled for 3 winding transformers, maybe you can just keep and rephrase slightly the comment to make it clearer.

Reply:
removed